agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -93,14 +93,26 @@ while true; do
 		;;
 	esac
 done
+
+function setup_ara {
```

Comment:
Why does this function exist here, and in `pbench-sysinfo-dump`?  Shouldn't it only live theere?